manifests/config.pp

```diff
@@ -35,4 +35,12 @@
     owner   => 'root',
     group   => 'tomcat',
   }
+
+  file { '/var/lib/tomcat/webapps/candlepin/WEB-INF/classes/broker.xml':
```

Comment:
We probably want to land our own broker.xml, which we can configure candlepin to read from!

```
candlepin.audit.hornetq.config_path=/etc/candlepin/broker.xml
```